roles/ansible_repositories/tasks/redhat.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: 'Install centos-release-ansible29 on CentOS8'
+  yum:
```

Comment:
Since this is targeted at EL8 OS should this use `dnf` ?

Reply:
Fair enough, I've updated it! :)